docs/containers/debug-netcore.md

```diff
@@ -60,3 +78,16 @@ To enable SSL (using the HTTPS protocol), you will need to make a few changes to
    ```
 
 For additional customization options, see the documentation on [Tasks](/docs/containers/reference.md) and [Debug containerized apps](/docs/containers/debug-common.md).
+
+## Saving Project File Preference for [.NET SDK Container Build](https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container)
+
+If you have a workspace folder with multiple .NET project files and you want to exclusively debug one specific project (without being prompted to choose from a list of project files every time you `kb(workbench.action.debug.start)`), you can save your launch profile by following these steps:
+
+1. Follow the steps in [.NET SDK Container Build](#net-sdk-container-build-debug-without-dockerfile) and keep the debug session live
+1. Click on the `gear` icon in your debugger view.
+
+   ![dockerSharedFolders](images/debug/debugger-scaffolding.png)
+
+1. Select **Docker: Debug in Container**
+1. Choose the project file associated with the project you want to debug
+1. Your project preference is saved & you no longer need to choose a project file on `kb(workbench.action.debug.start)`
```

Comment:
```suggestion
    Your project preference is saved, and you no longer need to choose a project file on `kb(workbench.action.debug.start)`
```

Reply:
Each step should be a user action. For other text, you can just indent it to align with the start of the previous paragraph (doublecheck the indentation)

Reply:
Makes sense